pages/common/awslogs.md

```diff
@@ -0,0 +1,24 @@
+# awslogs
+
+> Queries groups, streams and events from Amazon Cloudwatch logs.
+> More information: <https://github.com/jorgebastida/awslogs>.
+
+- List log groups:
+
+`awslogs groups`
+
+- List existing streams for specified group:
+
+`awslogs streams {{/var/log/syslog}}`
+
+- Get logs for any streams in the specified group between 1 and 2 hours ago:
+
+`awslogs get {{/var/log/syslog}} --start='{{2h ago}}' --end='{{1h ago}}'`
+
+- Get logs that match a specific Cloudwatch Logs Filter pattern:
+
+`awslogs get {{/aws/lambda/my_lambda_group}} --filter-pattern='ERROR'`
```

Comment:
I think ERROR should be in tokens(`{{}}`) too ?

Reply:
Ah yeah you're right @agnivade just added it!